src/commands/daily.ts

```
@@ -22,12 +23,17 @@ export const dailyCommand = define({
 			logger.level = 0;
 		}
 
+		// Create a single PricingFetcher instance to reuse across functions
+		using pricingFetcher = ctx.values.mode === 'display' ? undefined : new PricingFetcher(ctx.values.fetch);
```

ESLint

[error] Unsafe member access .values on an `any` value.

(ts/no-unsafe-member-access)